products/compute/inspec.yaml

```diff
@@ -32,6 +32,11 @@ overrides: !ruby/object:Provider::ResourceOverrides
     exclude: true
   DiskType: !ruby/object:Provider::Inspec::ResourceOverride
     exclude: true
+  Firewall: !ruby/object:Provider::Inspec::ResourceOverride
+    custom_code: |
```

Comment:
It looks like this is where the additional functions (`has_http_open`) are going to be located, right?

Could you rename this to something like `added_functions` or something like that?